pkg/analysis/passes/archivename/archivename_test.go

```diff
@@ -0,0 +1,59 @@
+package archivename
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/grafana/plugin-validator/pkg/analysis"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/archive"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/metadata"
+	"github.com/grafana/plugin-validator/pkg/testpassinterceptor"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const pluginId = "test-plugin-panel"
+
+func TestRootDirNotDist(t *testing.T) {
+	var interceptor testpassinterceptor.TestPassInterceptor
+
+	pass := &analysis.Pass{
+		RootDir: filepath.Join("testdata", "dist-folder"),
+		ResultOf: map[*analysis.Analyzer]interface{}{
+			archive.Analyzer:  filepath.Join("testdata", "dist-folder", "dist"),
+			metadata.Analyzer: []byte(`{"ID": "` + pluginId + `"}`),
+		},
+		Report: interceptor.ReportInterceptor(),
+	}
+
+	_, err := Analyzer.Run(pass)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Tip: you can change this to:
```suggestion
	require.NoError(t, err)
```